<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Aswinikumars-MacBook-Pro.local" name="twc.Automation.ExecutableTestCases.LATAM_CO_Run" tests="21" failures="0" timestamp="27 Mar 2022 14:33:20 GMT" time="424.256" errors="2">
  <testcase name="Verify_Lotame_Call_for_LATAM_CO" time="0.508" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Smoke_Test_CaseVerify_Video_adCall_LATAM_CO" time="0.290" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_Amazon_Call" time="0.315" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_WFXTriggers_Call_privacy_for_LATAM_CO" time="0.298" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_feed1_adcall_for_LATAM_CO" time="0.222" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="validate_Video_call_npa_val_for_LATAM_CO" time="0.389" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Smoke_Test_CaseVerify_Homescreen_marquee_adCall_LATAM_CO" time="0.268" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_Hourly_details_adcall_for_LATAM_CO" time="0.206" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_LocationWFXTriggers_Call__for_LATAM_CO" time="0.257" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_Hourly2_details_adcall_for_LATAM_CO" time="0.216" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_feed3_adcall_for_LATAM_CO" time="0.255" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_Hourly1_details_adcall_for_LATAM_CO" time="0.243" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_PreRollVideo_adcall_for_LATAM_CO" time="0.212" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_feed2_adcall_for_LATAM_CO" time="0.201" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_Criteo_SDK_config_app_Call_for_LATAM_CO" time="0.285" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at java.io.File.<init>(File.java:277)
	at twc.Automation.HandleWithCharles.CharlesFunctions.archive_folder(CharlesFunctions.java:318)
	at twc.Automation.ExecutableTestCases.LATAM_CO_Run.Verify_Criteo_SDK_config_app_Call_for_LATAM_CO(LATAM_CO_Run.java:357)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Verify_Criteo_SDK_config_app_Call_for_LATAM_CO -->
  <testcase name="preConditionsTest_for_LATAM_CO" time="418.786" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at java.io.File.<init>(File.java:277)
	at twc.Automation.HandleWithCharles.CharlesFunctions.archive_folder(CharlesFunctions.java:318)
	at twc.Automation.ExecutableTestCases.LATAM_CO_Run.preConditionsTest_for_LATAM_CO(LATAM_CO_Run.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- preConditionsTest_for_LATAM_CO -->
  <testcase name="Verify_Criteo_SDK_inapp_v2_Call_for_LATAM_CO" time="0.348" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_map_details_adcall_for_LATAM_CO" time="0.205" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_Daily_details_adcall_for_LATAM_CO" time="0.229" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="validate_NextGen_IM_call_npa_val_for_LATAM_CO" time="0.312" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
  <testcase name="Verify_amazon_aax_Hourly3_details_adcall_for_LATAM_CO" time="0.211" classname="twc.Automation.ExecutableTestCases.LATAM_CO_Run"/>
</testsuite> <!-- twc.Automation.ExecutableTestCases.LATAM_CO_Run -->
